test/global_uid_test.rb

```diff
@@ -394,15 +394,14 @@ def parent_child_fork_values
 
   private
 
-  def test_unique_ids
-    seen = {}
-    (0..10).each do
-      foo = WithGlobalUID.new
-      foo.save
-      refute_nil foo.id
-      assert_nil foo.description
-      refute seen.has_key?(foo.id)
-      seen[foo.id] = 1
+  def test_unique_ids(amount = 10)
+    amount.times.each_with_object({}) do |_, seen|
+      WithGlobalUID.create!.tap do |record|
```

Comment:
Do we need the `tap` here? I would understand if this was inside a `map` block, but inside `each_with_object` the return value of the block doesn’t matter. So perhaps just `record = WithGlobalUID.create!` here?

Reply:
You're right, we don't need the tap. The method went under a couple of iterations and it was left behind 😅